tasks/deploy.py

```diff
@@ -391,26 +391,21 @@ def deploy_built_artefacts(pr, event_info):
     """
     funcname = sys._getframe().f_code.co_name
 
-    log(f"{funcname}(): deploy for PR {pr.number}")
-
-    deploycfg = config.get_section(DEPLOYCFG)
-
     # verify that the GH account that set label bot:deploy has the
     # permission to trigger the deployment
```

Comment:
This comment should be moved down a bit, right above the `if labeler not in ...`